neqo-transport/src/crypto.rs

```diff
@@ -861,115 +861,132 @@ impl CryptoStreams {
     }
 
     pub fn send(&mut self, space: PNSpace, data: &[u8]) {
-        self[space].tx.send(data);
+        if let Some(cs) = self.get_mut(space) {
+            cs.tx.send(data);
+        } else {
+            panic!("Trying to send in a dropped space")
+        }
```

Comment:
Why not ....

```suggestion
        self.get_mut(space).unwrap().tx.send(data);
```
We don't need special panics, do we?  If you want the message, switch to `.expect("message")`